src/modules/inspect/process-list/process-list.ts

```diff
@@ -116,10 +111,25 @@ export class ProcessList {
     }
   }
 
-  public get remoteSolutionUri(): string {
-    const remoteSolutionUri: string = window.localStorage.getItem('processEngineRoute');
+  public async updateProcesses(): Promise<void> {
+    try {
+      const correlations: Array<Correlation> = await this._getCorrelations();
+      const correlationListWasUpdated: boolean = JSON.stringify(correlations) !== JSON.stringify(this._correlations);
+
+      if (correlationListWasUpdated) {
+        this._correlations = correlations;
+      }
+
+      this.succesfullRequested = true;
+    } catch (error) {
+      this._notificationService.showNotification(NotificationType.ERROR, `Error receiving process list: ${error.message}`);
+    }
+
+    if (!this._correlations) {
```

Comment:
Extract Business Rule

```suggestion
    const correlationNotSet: boolean = correlation === undefined;
    if (correlationNotSet) {
```